EpiJS/model.js

```diff
@@ -189,6 +189,7 @@ function mexport(model, output, file_type=".json") {
 		"key": model.key
 	}
 	for (let x in model.compartments) {
+		/* istanbul ignore if */
```

Comment:
Why is this being ignored?

Reply:
This can't be assured to run every time we run a test, thus is ignored.